app/src/main/java/com/ucmobiledevelopment/freeways/AppModule.kt

```diff
@@ -5,7 +5,7 @@ import org.koin.dsl.module
 
 @JvmField
 val AppModule = module {
-    viewModel { MainViewModel() }
+    viewModel { MainViewModel(get()) }
```

Comment:
I added this as per the request of the comment on line 10. If this is unnecessary or not what you meant, omit it please!

Reply:
It looks like the MVM constructor requires a parameter... the question is, does Koin know about the parameter type?  Is that also defined in Koin?